tests/redux/reducers/search/downloadReducer-test.js

```diff
@@ -0,0 +1,104 @@
+/**
+ * downloadReducer-test.js
+ * Created by Emily Gullo 08/24/2017
+ */
+
+import downloadReducer from 'redux/reducers/search/downloadReducer';
+import { List } from 'immutable';
+
+const initialState = {
```

Comment:
This can be imported from downloadReducer:
`import downloadReducer, { initialState } from 'redux/reducers/search/downloadReducer';`

Reply:
👍